### PR TITLE
(RE-4938) Add a LICENSE file and support info

### PR DESCRIPTION
**Pull request by dankreek**

### CODE REVIEW

**Comment by nwolfe** [comment]
This one's going to be tough to merge.

**Comment by jeffmccune** [comment]
> This one's going to be tough to merge.

How come?

**Comment by camlow325** [comment]
Needs rebase

**Comment by dankreek** [comment]
Have I alleviated your pain, @nwolfe?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (jeffmccune) [inline LICENSE]: Might be best to fill this in, or just copy the shorter version from Puppet's repo.

Comment (dankreek) [inline LICENSE]: GAH.. dang it.

**Pull request merged**